packages/untp-test-suite/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:20.12.2
+WORKDIR /app
+
+COPY . .
+
+RUN yarn install
+RUN yarn add @types/node @types/lodash @types/semver --dev
+RUN yarn add nodemon --dev
+
+RUN yarn build
+
+RUN yarn run build
+
+RUN yarn run untp
+
+RUN yarn run untp config
+
+RUN yarn run untp test
```

Comment:
Thank you for your contribution, I am a bit confused here, I don't think we need these commands. Should we remove them?

Reply:
@husamahmud @EmadAnwer if you want to answer in here